Model/Order/Buyer/Builder.php

```diff
@@ -52,13 +52,18 @@ class Builder extends PersonBuilder
      * @inheritdoc
      * @return Buyer
      */
-    public function buildObject(
+    // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
```

Comment:
Remove this, the `// @codingStandardsIgnoreLine` on the line below should do the trick